.gitignore

```diff
@@ -0,0 +1,5 @@
+#Ignor for binary files
+
+*.class
+gradle/wrapper/gradle-wrapper.jar
```

Comment:
The `gradle-wrapper.jar` is an exception from the rule "not to keep binaries in the repository" because it is the way `gradle` uses to allow running gradle commands without having a `gradle` installed, running all the command  through `gradlew`. For instance:
```bash
bash gradlew clean test
```
instead of:
```bash
gradle clean test
```
So, do not add this line to the `.gitignore` and keep it in the repository. All the `gradle` directory under the root of the project is better to be kept in the repository.